instance/admin.py

```diff
@@ -24,7 +24,7 @@
 
 from django.contrib import admin
 from django.contrib.contenttypes.admin import GenericStackedInline
-from django_extensions.db.fields.json import JSONField
+from django.contrib.postgres.fields import JSONField
```

Comment:
Won't we need a new migration that updates the fields for existing users of Ocim who've already run all of these migrations?

Reply:
Hi, @UmanShahzad. I'm going to move to a new migration scheme and test for it in new test cases. From what I can see, it shouldn't have any major difference between these two approaches, but architecturally, it makes sense to create a new migration. Thank you for pointing that out.

Reply:
If Postgres has to run an `ALTER TABLE` command to make the change internally, then we have no choice but to make a new migration for current users of Ocim.

Reply:
Hi, @UmanShahzad. I had to explicitly convert the VARCHAR/TEXT field into a JSON hstore through Postgresql, using the original field/column (just like described in the migration file). I'm finishing tests for that; once I have those finished, I'll let you know. Thanks.